### PR TITLE
rng shuffle to desc

### PR DESCRIPTION
**Pull request by shssf**
Depends on PR #800

### CODE REVIEW

File: dpnp/random/dpnp_iface_random.py [hunk 1]
```diff
@@ -1289,7 +1289,7 @@ def sample(size):
     return call_origin(numpy.random.sample, size)
 
 
-def shuffle(x):
+def shuffle(x1):
```
Comment (samir-nasibli) [inline dpnp/random/dpnp_iface_random.py:1292]: why `x1`?

**Review by samir-nasibli — APPROVED** [review]

File: dpnp/random/dpnp_iface_random.py (hunk 1 above)
Comment (shssf) [reply dpnp/random/dpnp_iface_random.py:1293]: Because one letter variable is worse solution

**Comment by shssf** [comment]
@samir-nasibli I suspect the problem is in test. Do you have any thoughts?

File: dpnp/random/dpnp_iface_random.py (hunk 1 above)
Comment (samir-nasibli) [reply dpnp/random/dpnp_iface_random.py:1293]: I see. It is just little bit confuse. I just thought `1` means `1` dimension (1 dimensional array) or something else.
But this is not critical. It is OK.

**Comment by samir-nasibli** [comment]
@shssf 
There were failed tests:
```bash
tests/test_random.py::TestPermutationsTestShuffle::test_shuffle1_fallback[ lambda x: numpy.asarray(x).astype(numpy.float32)]
tests/test_random.py::TestPermutationsTestShuffle::test_shuffle1_fallback[ lambda x: numpy.asarray([[i, i] for i in x])]
tests/test_random.py::TestPermutationsTestShuffle::test_shuffle1_fallback[ lambda x: numpy.vstack([x, x]).T]
```
I fixed tests.
Tests failed because `numpy` callback did not occur in the specified test cases as expected in the tests `tests/test_random.py::TestPermutationsTestShuffle::test_shuffle1_fallback`.
Similar test cases do not fall in `tests/test_random.py::TestPermutationsTestShuffle::test_shuffle1`.
See:
``` bash
tests/test_random.py::TestPermutationsTestShuffle::test_shuffle1[lambda x: dpnp.asarray(x).astype(dpnp.float32)] PASSED
tests/test_random.py::TestPermutationsTestShuffle::test_shuffle1[lambda x: dpnp.asarray([[i, i] for i in x])] PASSED
tests/test_random.py::TestPermutationsTestShuffle::test_shuffle1[lambda x: dpnp.vstack([x, x]).T] PASSED 
```

Initially it was wrong to use `numpy` data in `dpnp` own tests.

Now I have changed the testing logic, removed the `numpy` data and leave only that test cases for `tests/test_random.py::TestPermutationsTestShuffle::test_shuffle1_fallback` what has not yet been implemented in dpnp implementation (handling with given lists, for example).

Changes could be moved to a separate RP, but I decided to leave everything here.
Thank you!

**Pull request merged**